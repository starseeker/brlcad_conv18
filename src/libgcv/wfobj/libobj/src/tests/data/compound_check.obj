g g_EMPTY1 gEMPTY1 # should be 1 empty group, tagging for elements only
o o_bad # should be ignored, object taggingis for elements only
shadow_obj s_bad
trace_obj t_bad
lod 1
s 1
bevel on
c_interp on
d_interp on
v 0 0 0
v 0 1 0
v 1 1 0
v 1 0 0

g g_EMPTY2
lod 100
s 1
bevel off
c_interp off
d_interp off
vt 10 10 10
vt 10 11 10
vt 11 11 10
vt 11 10 10

lod 0
s 0
bevel off
c_interp off
d_interp off
vn 20 20 20
vn 20 21 20
vn 21 21 20
vn 21 20 20

g g_a g_b g_c
o o_1
shadow_obj so_1
trace_obj to_1
lod 1
s 1
mtllib mtllib_foo1 mtllib_bar1
usemtl mtl_foo
maplib maplib_foo1 maplib_bar1
usemap map_foo
bevel on
c_interp on
d_interp on
p 1 2 3
l 1 2 3
f 1 2 3

g g_a
o o_2
shadow_obj so_2
trace_obj to_2
lod 3
lod 2
s 3
s 4
mtllib mtllib_foo2 mtllib_bar2
usemtl mtl_bar
maplib maplib_foo2 maplib_bar2
usemap map_bar
bevel on
bevel off
c_interp on
c_interp off
d_interp on
d_interp off
p 1 3 4
l 1/1 3/3 4/4
f 1/1/1 3/3/3 4/4/4

usemap off
f 1 4 3

o o_bad2 # should be ignored, object taggingis for elements only
shadow_obj so_bad2
trace_obj to_bad2
v 30 30 31
v 30 31 31
v 31 31 31
v 31 30 31

vt 40 40 41
vt 40 41 41
vt 41 41 41
vt 41 40 41

vn 50 50 51
vn 50 51 51
vn 51 51 51
vn 51 50 51

mtllib mtllib_ignore1 mtllib_ignore2
mtllib mtllib_ignore3 mtllib_ignore2
usemtl mtl_ignore1
usemtl mtl_ignore2
maplib maplib_ignore1 maplib_ignore2
maplib maplib_ignore3 maplib_ignore2
usemap map_ignore1
usemap map_ignore2

g g_d g_e g_f
g g_h g_i g_j
o o_1 # should be same as v1
shadow_obj so_1
trace_obj to_1
lod 0
s off
mtllib mtllib_bar1 mtllib_foo1 #same as face 1
usemtl mtl_foo
maplib maplib_bar1 maplib_foo1 #same as face 1
usemap map_foo
bevel off
c_interp off
d_interp off
p 5 6 7
l 5/5 6/6 7/7
f 5/5 6/6 7/7

g g_e
o o_ignore1
o o_ignore2
o o_3
shadow_obj so_ignore1
shadow_obj so_ignore2
shadow_obj o_3
trace_obj to_ignore1
trace_obj to_ignore2
trace_obj to_3
lod 1
s 1
mtllib mtllib_foobar
usemtl mtl_foobar
maplib maplib_foobar
usemap map_foobar
bevel on
c_interp on
d_interp on
p 5 7 8
l 5 7 8
f 5//5 7//7 8//8
