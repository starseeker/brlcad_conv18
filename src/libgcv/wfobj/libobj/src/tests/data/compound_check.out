vertices {
  <0,0,0,1>
  <0,1,0,1>
  <1,1,0,1>
  <1,0,0,1>
  <30,30,31,1>
  <30,31,31,1>
  <31,31,31,1>
  <31,30,31,1>
}
texture coordinates {
  <10,10,10>
  <10,11,10>
  <11,11,10>
  <11,10,10>
  <40,40,41>
  <40,41,41>
  <41,41,41>
  <41,40,41>
}
normals {
  <20,20,20>
  <20,21,20>
  <21,21,20>
  <21,20,20>
  <50,50,51>
  <50,51,51>
  <51,51,51>
  <51,50,51>
}
point {
  groups: g_a g_b g_c
  object: 'o_1'
  material: 'mtl_foo'
  materiallibs: mtllib_bar1 mtllib_foo1
  texmap: 'map_foo'
  texmaplibs: maplib_bar1 maplib_foo1
  shadow_obj: 'so_1'
  trace_obj: 'to_1'
  smoothing group: '1'
  bevel: '1'
  c_interp: '1'
  d_interp: '1'
  lod: '1'
  vertices: 1 2 3 
}
point {
  groups: g_a
  object: 'o_2'
  material: 'mtl_bar'
  materiallibs: mtllib_bar2 mtllib_foo2
  texmap: 'map_bar'
  texmaplibs: maplib_bar2 maplib_foo2
  shadow_obj: 'so_2'
  trace_obj: 'to_2'
  smoothing group: '4'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '2'
  vertices: 1 3 4 
}
point {
  groups: g_h g_i g_j
  object: 'o_1'
  material: 'mtl_foo'
  materiallibs: mtllib_bar1 mtllib_foo1
  texmap: 'map_foo'
  texmaplibs: maplib_bar1 maplib_foo1
  shadow_obj: 'so_1'
  trace_obj: 'to_1'
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5 6 7 
}
point {
  groups: g_e
  object: 'o_3'
  material: 'mtl_foobar'
  materiallibs: mtllib_foobar
  texmap: 'map_foobar'
  texmaplibs: maplib_foobar
  shadow_obj: 'o_3'
  trace_obj: 'to_3'
  smoothing group: '1'
  bevel: '1'
  c_interp: '1'
  d_interp: '1'
  lod: '1'
  vertices: 5 7 8 
}
line {
  groups: g_a g_b g_c
  object: 'o_1'
  material: 'mtl_foo'
  materiallibs: mtllib_bar1 mtllib_foo1
  texmap: 'map_foo'
  texmaplibs: maplib_bar1 maplib_foo1
  shadow_obj: 'so_1'
  trace_obj: 'to_1'
  smoothing group: '1'
  bevel: '1'
  c_interp: '1'
  d_interp: '1'
  lod: '1'
  vertices: 1 2 3 
}
line {
  groups: g_e
  object: 'o_3'
  material: 'mtl_foobar'
  materiallibs: mtllib_foobar
  texmap: 'map_foobar'
  texmaplibs: maplib_foobar
  shadow_obj: 'o_3'
  trace_obj: 'to_3'
  smoothing group: '1'
  bevel: '1'
  c_interp: '1'
  d_interp: '1'
  lod: '1'
  vertices: 5 7 8 
}
textured line {
  groups: g_a
  object: 'o_2'
  material: 'mtl_bar'
  materiallibs: mtllib_bar2 mtllib_foo2
  texmap: 'map_bar'
  texmaplibs: maplib_bar2 maplib_foo2
  shadow_obj: 'so_2'
  trace_obj: 'to_2'
  smoothing group: '4'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '2'
  vertices: 1/1 3/3 4/4 
}
textured line {
  groups: g_h g_i g_j
  object: 'o_1'
  material: 'mtl_foo'
  materiallibs: mtllib_bar1 mtllib_foo1
  texmap: 'map_foo'
  texmaplibs: maplib_bar1 maplib_foo1
  shadow_obj: 'so_1'
  trace_obj: 'to_1'
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5/5 6/6 7/7 
}
face {
  groups: g_a g_b g_c
  object: 'o_1'
  material: 'mtl_foo'
  materiallibs: mtllib_bar1 mtllib_foo1
  texmap: 'map_foo'
  texmaplibs: maplib_bar1 maplib_foo1
  shadow_obj: 'so_1'
  trace_obj: 'to_1'
  smoothing group: '1'
  bevel: '1'
  c_interp: '1'
  d_interp: '1'
  lod: '1'
  vertices: 1 2 3 
}
face {
  groups: g_a
  object: 'o_2'
  material: 'mtl_bar'
  materiallibs: mtllib_bar2 mtllib_foo2
  texmap: ''
  texmaplibs: maplib_bar2 maplib_foo2
  shadow_obj: 'so_2'
  trace_obj: 'to_2'
  smoothing group: '4'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '2'
  vertices: 1 4 3 
}
textured face {
  groups: g_h g_i g_j
  object: 'o_1'
  material: 'mtl_foo'
  materiallibs: mtllib_bar1 mtllib_foo1
  texmap: 'map_foo'
  texmaplibs: maplib_bar1 maplib_foo1
  shadow_obj: 'so_1'
  trace_obj: 'to_1'
  smoothing group: '0'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '0'
  vertices: 5/5 6/6 7/7 
}
oriented face {
  groups: g_e
  object: 'o_3'
  material: 'mtl_foobar'
  materiallibs: mtllib_foobar
  texmap: 'map_foobar'
  texmaplibs: maplib_foobar
  shadow_obj: 'o_3'
  trace_obj: 'to_3'
  smoothing group: '1'
  bevel: '1'
  c_interp: '1'
  d_interp: '1'
  lod: '1'
  vertices: 5//5 7//7 8//8 
}
textured oriented face {
  groups: g_a
  object: 'o_2'
  material: 'mtl_bar'
  materiallibs: mtllib_bar2 mtllib_foo2
  texmap: 'map_bar'
  texmaplibs: maplib_bar2 maplib_foo2
  shadow_obj: 'so_2'
  trace_obj: 'to_2'
  smoothing group: '4'
  bevel: '0'
  c_interp: '0'
  d_interp: '0'
  lod: '2'
  vertices: 1/1/1 3/3/3 4/4/4 
}
